### DISCUSSION TITLE
Automazione si riavvia a mezzanotte..

### DISCUSSION THREAD

**Discussion by Steefano72** [indomus/forum#3013] [GENERALE - Discussioni generiche]
Ciao a tutti,
Ho creato una automazione semplicissima che accende le luci esterne al tramonto e le spegne alle 2:00 di notte e funziona bene. Pero' se, ad esempio,  spengo manualmente quelle luci alle 23:00, queste si riaccendono alle 24:00. Sto sbagliando qualcosa ? Come posso risolvere?
Grazie
Stefano

**Comment by snakuzzo** [comment]
perdonami, ma uno come può dirti se sbagli qualcosa se non fai vedere la tua automazione?
Pubblica qui il codice YAML **usando il markdown**

**Comment by Steefano72** [comment]
Scusami ho fatto un po' di confusione..
ho utilizzato l'integrazione scheduler v3.2.12 quindi nel file automations.yaml  non lo vedo.  Dove trovo il codice YAML in questo caso?
grazie

**Reply by snakuzzo** [reply]
Innanzi tutto, a meno che non si tratti di nuovi thread, non rispondere creando nuovi thread (come hai appena fatto), ma rispondi sotto l'ultimo messaggio.

Quanto a quello che hai scritto...che cosa sarebbe questa integrazione scheduler? E' un custom component ?

**Reply by Steefano72** [reply]
ok.. non sono molto pratico. 
Sì è un custom component
